tests/sqlite_integration_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2025 Google LLC
```

Comment:
```suggestion
//go:build integration && sqlite

// Copyright 2025 Google LLC
```

We'll also have to add `sqlite` to https://github.com/googleapis/genai-toolbox/blob/main/.golangci.yaml